src/commands/general/yourcontrols.ts

```diff
@@ -0,0 +1,11 @@
+import { CommandDefinition } from '../../lib/command';
+import { CommandCategory } from '../../constants';
+
+export const yourcontrols: CommandDefinition = {
+    name: ['yourcontrols', 'yc'],
+    description: 'Provides link to the YourControls Discord server',
+    category: CommandCategory.GENERAL,
+    executor: async (msg) => {
+        await msg.channel.send('https://discord.gg/p7Bzcv3Yjd');
```

Comment:
```suggestion
        return msg.channel.send('https://discord.gg/p7Bzcv3Yjd');
```

Reply:
Is this a link you know won't get deleted? (as in talked to admins so they don't prune it)

Reply:
> Is this a link you know won't get deleted? (as in talked to admins so they don't prune it)

This is the link they have on their github